packages/components/datepicker/stories/Calendar.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+
+import { Calendar } from '../src';
+import type { CalendarProps } from '../src';
+
+export default {
+  component: Calendar,
+  title: 'Components/Datepicker/Calendar',
+} as Meta;
+
+export const Default: Story<CalendarProps> = (args) => {
+  const [selectedDay, setSelectedDay] = useState<Date>(new Date());
+
+  return (
+    <Calendar
+      {...args}
+      mode="single"
+      selected={selectedDay}
+      onSelect={setSelectedDay}
+    />
+  );
+};
+
+export const WithFromToDates: Story<CalendarProps> = (args) => {
+  const [selectedDay, setSelectedDay] = useState<Date>(new Date());
+
+  return (
+    <Calendar
+      {...args}
+      mode="single"
```

Comment:
Do we want to support other modes, or should we omit this prop from `CalendarProps`?

Reply:
I would support it, don't see any reason to restrict it.

Reply:
Nevermind, I see you use both.